src/annotator/config/settings.js

```diff
@@ -102,9 +102,22 @@ function configFuncSettingsFrom(window_) {
   return window_.hypothesisConfig();
 }
 
+/**
+ * Return true if the client is from a browser extension.
+ *
+ * @returns {boolean} true if this instance of the Hypothesis client is one
+ *   distributed in a browser extension, false if it's one embedded in a
+ *   website.
+ *
+ */
+function isBrowserExtension(config) {
+  return !(config.app.startsWith('http://') || config.app.startsWith('https://'));
```

Comment:
looks like ie11 doesn't support this function. perhaps we need to add a polyfill for it or do the indexOf check?